src/lib/requestManager.ts

```diff
@@ -14,17 +14,19 @@ const defaultOptions: { responseType: ResponseType } = {
  *                   an error object for its reason
  */
 
-function requestManager(method: HTTPMethod, endpoint: string, requestOptions: AxiosRequestConfig = {}) {
+function requestManager(
+  method: HTTPMethod,
+  endpoint: string,
+  requestOptions: AxiosRequestConfig = {}
+): Promise<AxiosResponse> {
   const requestParams: AxiosRequestConfig = {
     method,
     url: endpoint,
     ...defaultOptions,
     ...requestOptions
   }
 
-  return axios.request(requestParams).then((response: AxiosResponse<any>) => {
-    return response.data
-  })
+  return axios.request(requestParams)
```

Comment:
Is it necessary to change this implementation?

Reply:
Yes, because the previous implementation returned only data but not status code and I need the check the status code in the handler.